core/sql/sqlcomp/CmpDescribe.cpp

```diff
@@ -4129,6 +4219,33 @@ char buf[1000];
        ((CmpCommon::getDefault(SHOWDDL_DISPLAY_PRIVILEGE_GRANTS) == DF_SYSTEM)
            && getenv("SQLMX_REGRESS"))) ? FALSE : TRUE;
 
+//display library comment
+   if (libraryUID > 0)
+   {
+     if (cmpSBD.switchCompiler())
+       {
+         *CmpCommon::diags() << DgSqlCode(-CAT_UNABLE_TO_RETRIEVE_PRIVS);
+         return -1;
+       }
+
+     ComTdbVirtObjCommentInfo * objCommentInfo = NULL;
+     cmpSBD.getSeabaseObjectComment(libraryUID, COM_LIBRARY_OBJECT, objCommentInfo);
```

Comment:
getSeabaseObjectComment does return an error.  Also, it looks like objCommentInfo is null if unable to retrieve comment.  However, did you want to report the issue or just not display comments.  At a minimum you could say "-- comment details are not available" and/or log the failure as an INFO message.  In getSeabaseObjectComment, some errors are placed in ComDiags and some are not.  Will the error be returned as part of showddl output if ComDiags is setup?